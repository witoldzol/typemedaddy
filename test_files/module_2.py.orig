def foo(i):
    return i

def bar(x):
    return x
