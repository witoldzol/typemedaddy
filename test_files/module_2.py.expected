def foo(i: int) -> int:
    return i

def bar(x: str) -> str:
    return x
